File: app/src/main/java/ga/forntoh/bableschool/model/misc/TopSchool.java
package ga.forntoh.bableschool.model.misc;

import ga.forntoh.bableschool.utils.Utils;

@SuppressWarnings("unused")
public class TopSchool {

    private String image, schoolName, topStudentName;
    private double average;

    public String getImage() {
        return image;
    }

    public String getSchoolName() {
        return schoolName;
    }

    public String getTopStudentName() {
        return Utils.capEachWord(topStudentName);
    }

    public double getAverage() {
        return average;
    }
}

File: app/src/main/java/ga/forntoh/bableschool/data/model/other/TopSchool.kt
package ga.forntoh.bableschool.data.model.other

import androidx.room.ColumnInfo
import androidx.room.Entity

@Entity(primaryKeys = ["schoolName", "topStudentName"])
data class TopSchool(
        @ColumnInfo var schoolName: String = "",
        @ColumnInfo var topStudentName: String = "",
        @ColumnInfo var image: String? = "",
        @ColumnInfo var average: Double = 0.toDouble()
)